content/common/view_messages.h

```diff
@@ -174,6 +174,7 @@ IPC_STRUCT_TRAITS_BEGIN(blink::WebScreenInfo)
   IPC_STRUCT_TRAITS_MEMBER(availableRect)
   IPC_STRUCT_TRAITS_MEMBER(orientationType)
   IPC_STRUCT_TRAITS_MEMBER(orientationAngle)
+  IPC_STRUCT_TRAITS_MEMBER(deviceRadius) //[device-radius]
```

Comment:
Okay, I will progress the work in this repository after applying our blink changes to the blink-crosswalk repository.